### ISSUE TITLE
Error on french description

### ISSUE THREAD

**Issue by primael** [jmrozanec/cron-utils#338]
When we use a quartz expression like `* * * * * ? *`, we expect `chaque seconde` as french description, but we have `chaque second`.

```       @Test
	public void testEverySecondInFrench() {
		CronParser cronParser = new CronParser(CronDefinitionBuilder.instanceDefinitionFor(CronType.QUARTZ));
		String cronString = "* * * * * ? *";
		Cron cron = cronParser.parse(cronString);
		String description = CronDescriptor.instance(Locale.FRANCE).describe(cron);
		Assert.assertEquals("chaque seconde", description);
	}
```

**Comment by jmrozanec** [comment]
@primael this is solved with PR #337. May we close it? If you require a release soon, just let us know! :smile:

**Comment by primael** [comment]
I've an error with sonar : 
`The command "if [ "$TRAVIS_PULL_REQUEST" != "false" ];then 
   mvn org.jacoco:jacoco-maven-plugin:prepare-agent package --settings target/travis/settings.xml -Dsettings.security=target/travis/settings-security.xml -Dmaven.javadoc.skip=true;
   sonar-scanner
fi
" exited with 127.
0.00s$ if [ "$TRAVIS_PULL_REQUEST" = "false" ];then 
   mvn org.jacoco:jacoco-maven-plugin:prepare-agent package deploy --settings target/travis/settings.xml -Dsettings.security=target/travis/settings-security.xml -Dmaven.javadoc.skip=true;
   sonar-scanner
fi****` 
any help would be appreciate. Pleaseeee
